.gitignore

```diff
@@ -72,9 +72,9 @@ _build
 /bytecomp/opcodes.ml
 /bytecomp/opcodes.mli
 
-/debugger/lexer.ml
-/debugger/parser.ml
-/debugger/parser.mli
+/debugger/debugger_lexer.ml
```

Comment:
I would prefer if the `Lexer => Debugger_lexer` change was made in a first, separate commit.

Reply:
I will split the commit once the timeline is stabilized.

Reply:
History has been rewritten and the commit split.